COMPILING.md

```diff
@@ -29,6 +29,11 @@ We assume that you have a Debian/Ubuntu or Red Hat-like distribution.
    ```
    Note that you need g++ version 5.0 or newer.
 
+   On Amazon Linux and similar distributions, do as root:
```

Comment:
Explicitly note this is ~= RHEL and therefore presumably Fedora too?

Reply:
No, these now got dnf